packages/e2e/cypress/integration/common/fileuploader.ts

```diff
@@ -0,0 +1,12 @@
+import { Then, When } from '@badeball/cypress-cucumber-preprocessor';
+
+When('I select a file with file name {string}', (fileName: string) => {
+  cy.get('input[type=file]').selectFile(
+    {
+      contents: Cypress.Buffer.from('file contents'),
+      fileName,
+      lastModified: Date.now(),
+    },
+    { force: true }
```

Comment:
Is this force necessary? If so, can we add a short comment why it's necessary?

e.g. https://github.com/aws-amplify/amplify-ui/blob/c11de75eaf0e0476cea56d7cd2da1ffa04d1c639/packages/e2e/cypress/integration/common/shared.ts#L178-L184

Reply:
I imagine this is because this element is hidden -- a comment explaining that would be helpful for future readers!

Reply:
Yup, I'll add that, and it is necessary